server/run.go

```diff
@@ -0,0 +1,68 @@
+package server
+
+import (
+	"context"
+	"io"
+	"net/http"
+
+	"github.com/benchttp/runner/config"
+	"github.com/benchttp/runner/internal/configparse"
+	"github.com/benchttp/runner/output"
+	"github.com/benchttp/runner/requester"
+)
+
+type runHandler struct{}
+
+func (h runHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/run" {
+		http.NotFound(w, r)
+		return
+	}
+
+	b, err := io.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	cfg, err := configparse.JSON(b)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	report, err := doRun(cfg)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	w.Write([]byte(report.String()))
+}
+
+func doRun(cfg config.Global) (output.Report, error) {
+	// Retrieve HTTP request generated by the config
+	req, err := cfg.Request.Value()
+	if err != nil {
+		return output.Report{}, err
+	}
+
+	bk, err := requester.New(requesterConfig(cfg)).Run(context.Background(), req)
```

Comment:
I think it's "bk" for "benchmark", because we used this term a lot in v0?
We'll have to change it then, could have been in this PR, but we can do it in one of the following ones too!

Reply:
Just saw that it is removed later in the PR!

Reply:
Yup agreed, we even explicitly decided to stop using that term (and removed it from our glossary).
So it's definitely going to be removed, but I'd rather do that in the dedicated refactoring PR (#9)